pkg/query/querier.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/go-kit/kit/log"
 	"github.com/opentracing/opentracing-go"
 	"github.com/pkg/errors"
+	ef "github.com/prometheus-community/prom-label-proxy/injectproxy"
```

Comment:
`ef` sounds too abstract, can you leave the import as it is?

Reply:
agree that ef is too abstract, but let's relabel it else it will be used as injectproxy and IMO it's kind of misleading because it is no longer used as a proxy. WDYT :)